cmd/version.go

```diff
@@ -86,7 +87,8 @@ func newVersionCmd() *cobra.Command {
 		},
 	}
 
-	versionCmdDescription := fmt.Sprintf("Output format to use: %s", outputFormatOptions)
+	versionCmdDescription := fmt.Sprintf("Output format. Allowed values: %s",
```

Comment:
Is `outputFormatOptions` superfluous? Should we use `getVersionsOutputFormatOptions` everywhere?

Reply:
`outputFormatOptions` came first--I didn't use one var because I wasn't sure the two commands would always support the same output formats.